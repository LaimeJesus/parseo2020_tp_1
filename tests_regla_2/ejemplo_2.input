fun f A -> B
fun c C -> D
