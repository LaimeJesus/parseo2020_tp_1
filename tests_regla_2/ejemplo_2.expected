[
   "program",
   [
      [
         "fun",
         "f",
         [
            "sig",
            [
               "_"
            ],
            "_"
         ],
         [
            "pre",
            [
               "true"
            ]
         ],
         [
            "post",
            [
               "true"
            ]
         ],
         [
            [
               "rule",
               [
                  [
                     "pcons",
                     "A",
                     []
                  ]
               ],
               [
                  "cons",
                  "B",
                  []
               ]
            ]
         ]
      ],
      [
         "fun",
         "c",
         [
            "sig",
            [
               "_"
            ],
            "_"
         ],
         [
            "pre",
            [
               "true"
            ]
         ],
         [
            "post",
            [
               "true"
            ]
         ],
         [
            [
               "rule",
               [
                  [
                     "pcons",
                     "C",
                     []
                  ]
               ],
               [
                  "cons",
                  "D",
                  []
               ]
            ]
         ]
      ]
   ],
   []
]
